lib/src/widgets/overlay.dart

```diff
@@ -125,7 +125,7 @@ class DescribedFeatureOverlay extends StatefulWidget {
     this.enablePulsingAnimation = true,
     this.allowShowingDuplicate = false,
     this.overflowMode = OverflowMode.ignore,
-    this.dismissOnTouchOutside = true,
+    this.barrierDismissible = true,
```

Comment:
@xsahil03x If you set a default value to this parameter, then this becomes a breaking change. Could you update the changelog and increase the version number?

Reply:
@axel-op this will not be a breaking change because the default value of `barrierDismissible` is allowing the users to dismiss the overlay by clicking outside. Which is similar to the previous behavior which means the `onDismiss` will work fine.

Reply:
Yeah but as you said:

> Also, throw an assertion if onDismiss and barrierDismissible are used at the same time

Reply:
Ok I agree this is will not really "break" anything, but I think we should at least let the users be aware of that